app/build.gradle

```diff
@@ -50,6 +52,9 @@ model {
         create("all") {
         }
     }
+    android.lintOptions {
```

Comment:
No doubts about that, currently there are few errors, specifically `NewApi` (SinkholeService.java:268) and `StrictFormatInvalid` (ActivityForward.java:63), others are *just warnings*